### PR TITLE
Add schemas back into vsix and remove iproj schema

### PR DESCRIPTION
**Pull request by worksofliam**
### Changes

Looks like the schema defs weren't being put into the vsix because we were just ignoring the entire `src` directory. This will add it back, which will fix the content assist not working for local the `actions.json` file.

This PR also removes the `iproj` schema as it will be moved to another extension. Code for IBM i doesn't really use `iproj` for any user interaction other than one place in the Action runner.

### Checklist

* [x] have tested my change
* [ ] updated relevant documentation
* [ ] Remove any/all `console.log`s I added
* [ ] eslint is not complaining
* [ ] have added myself to the contributors' list in [CONTRIBUTING.md](https://github.com/halcyon-tech/vscode-ibmi/blob/master/CONTRIBUTING.md)
* [ ] **for feature PRs**: PR only includes one feature enhancement.

### CODE REVIEW

**Review by sebjulliand — CHANGES_REQUESTED** [review]
The change is good, no questions asked 👍🏻 

However I'd be in favor of moving the `schemas` folder out of `src` (like `snippets`) and move `Objects.ts` and `LocalLanguageActions.ts` elsewhere, each in a different, meaningful location. Thoughts?

**Comment by worksofliam** [comment]
@sebjulliand great idea. I will move the schema and .ts files before requesting your review again. 

Thanks!

File: package.json [hunk 1]
```diff
@@ -49,13 +49,7 @@
...
 					"actions.json"
 				],
-				"url": "./src/schemas/actions.json"
-			},
-			{
-				"fileMatch": [
-					"iproj.json"
-				],
-				"url": "./src/schemas/iproj.json"
+				"url": "./snippets/actions.json"
```
Comment (sebjulliand) [inline package.json:52]: Wrong location
```suggestion
				"url": "./schemas/actions.json"
```

**Review by sebjulliand — CHANGES_REQUESTED** [review]
Almost OK. You just need to fix the path on line 43 to be `"./schemas/cl.code-snippets"`.

**Comment by worksofliam** [comment]
I need to fix the conflict first. Whoops.

**Comment by worksofliam** [comment]
@sebjulliand Ready for review again. Thanks!

**Review by sebjulliand — APPROVED** [review]
All good now, let's merge!

**Pull request merged**